### PR TITLE
Raspberry pi serial console

### PR DESCRIPTION
**Pull request by haggaie**
Make debugging a raspberry pi simpler by adding a serial console service and matching kernel command line parameters.

Tried to follow the suggestions in this older pull request: https://github.com/OpenELEC/OpenELEC.tv/pull/4161

### CODE REVIEW

**Comment by sraue** [comment]
please add the same way for RPi2 project too please

File: scripts/mkimage [hunk 1]
```diff
@@ -125,11 +125,11 @@ PROMPT 1
 
 LABEL installer
   KERNEL /$KERNEL_NAME
-  APPEND boot=UUID=$UUID_SYSTEM installer quiet tty vga=current
+  APPEND boot=UUID=$UUID_SYSTEM installer quiet tty vga=current $EXTRA_CMDLINE
```
Comment (stefansaraev) [inline scripts/mkimage]: x86 does not care.

File: scripts/mkimage [hunk 2]
```diff
@@ -125,11 +125,11 @@ PROMPT 1
...
 
 LABEL live
   KERNEL /$KERNEL_NAME
-  APPEND boot=UUID=$UUID_SYSTEM live quiet tty vga=current
+  APPEND boot=UUID=$UUID_SYSTEM live quiet tty vga=current $EXTRA_CMDLINE
```
Comment (stefansaraev) [inline scripts/mkimage]: here too.

**Comment by haggaie** [comment]
Hi,
I've updated the patches according to your comments. I don't have an RPi2 to test it though. From what I read online, /dev/ttyAMA0 should also work there.

**Comment by sraue** [comment]
please use master branch then openelec-6.0 branch for the PR

**Comment by sraue** [comment]
added in https://github.com/OpenELEC/OpenELEC.tv/commit/a09ae66ee443c6fcdba032542b905abe15c700dd https://github.com/OpenELEC/OpenELEC.tv/commit/e3203be00ea0a055f51da8760a37bcd2bcc17a7a and https://github.com/OpenELEC/OpenELEC.tv/commit/f90b396357e87757ef79a7dd772e80af95a16290

**Pull request closed without merging**